tests/frontend/matcher/test_twoway_withoutratiotest_matcher.py

```diff
@@ -10,31 +10,58 @@
 from gtsfm.frontend.matcher.twoway_matcher import TwoWayMatcher
 
 
-class TestTwoWayMatcher(test_matcher_base.TestMatcherBase):
-    """Unit tests for the TwoWayMatcher.
+class TestTwoWayWithoutRatioTestMatcher(test_matcher_base.TestMatcherBase):
+    """Unit tests for the TwoWayMatcher configured to not use the ratio-test.
 
     All unit test functions defined in TestMatcherBase are run automatically.
     """
 
     def setUp(self):
         super().setUp()
 
-        self.matcher = TwoWayMatcher()
+        self.matcher = TwoWayMatcher(ratio_test_threshold=None)
 
     def test_on_dummy_data(self):
         """Test using dummy 1D descriptors to verify correctness."""
 
-        image_shape_i1 = (300, 100) # as (H,W)
+        image_shape_i1 = (300, 100)  # as (H,W)
```

Comment:
why these formatting changes in this file? are your black settings consistent?

Reply:
I am not using any specific black formatter setting except the line length to 120.

From the PEP 8 guidelines, it seems like 2 spaces before the inline comment is the recommendation: https://www.python.org/dev/peps/pep-0008/#inline-comments